### PR TITLE
Changed information about Mopify

### PR DESCRIPTION
**Pull request by dirkgroenen**
Changed Mopify's information since the new beta just launched.

### CODE REVIEW

**Comment by kingosticks** [comment]
Doesn't your image path need updating to reflect it's changed to a jpg? Shame travis can't spot that.

**Comment by dirkgroenen** [comment]
Thought I already did that, but clearly I didn't; Just changed it.

**Comment by dirkgroenen** [comment]
Btw, it looks like Travis can't read some of the image files:
```shell
/home/travis/build/mopidy/mopidy/docs/ext/web.rst:None: WARNING: image file not readable: ext/moped.jpg
/home/travis/build/mopidy/mopidy/docs/ext/web.rst:None: WARNING: image file not readable: ext/mopify.png
```

**Comment by dirkgroenen** [comment]
Any progress on updating this page?

**Comment by kingosticks** [comment]
This one probably slipped through the cracks. I think the wording might need a slight tweak so it reads a little better, something like the following?

A web client that uses external services to provide additional features and a more 'complete' Spotify music experience. It's currently targeted at people using Spotify through Mopidy. Made by Dirk Groenen.

**Comment by dirkgroenen** [comment]
Jup, sounds good to me. I'll change it.

File: docs/ext/web.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ https://github.com/martijnboland/moped
 
 A Mopidy web client made with AngularJS by Martijn Boland.
 
-.. image:: /ext/moped.png
+.. image:: /ext/moped.jpg
```
Comment (jodal) [inline docs/ext/web.rst]: You're changing the file extension of the wrong image.

**Comment by jodal** [comment]
Can you shrink the image a bit? Currently its 240 kB, and will be included in all distributions of Mopidy as part of the docs, not just on some random web page.

**Comment by dirkgroenen** [comment]
I've just scaled the image which results in a size around 100kB. Also corrected moped's extension and fixed mopify's.

**Comment by jodal** [comment]
Amended a bit and merged in 7fd0603e160fa236f4de6b23df14d8164b854603.

**Pull request closed without merging**